.github/workflows/tests.yaml

```diff
@@ -51,7 +65,7 @@ jobs:
       BUNDLE_GEMFILE: "${{ matrix.gemfile }}"
       ALLOW_FAILURES: "${{ matrix.allow_failures }}"
     runs-on: ${{ matrix.os }}-latest
-    continue-on-error: ${{ endsWith(matrix.ruby, 'head') || matrix.ruby == 'debug' }}
+    continue-on-error: ${{ endsWith(matrix.ruby, 'head') || matrix.ruby == 'debug' || matrix.ruby == 'jruby' }}
```

Comment:
Is this needed? No test failure in jruby jobs

Reply:
Oh, I forgot delete this condition. thanks!
I fixed.
https://github.com/AssetSync/asset_sync/pull/444/commits/a1710ddba21d367eb9174e28aff59b891ff8e9c5

Reply:
@PikachuEXE